cfgov/v1/util/filterable_list.py

```diff
@@ -94,6 +97,11 @@ def process_form(self, request, form):
         filter_data['form'] = form
         return filter_data
 
+    def set_do_not_index(self, field, value):
+        """ Do not index on params except topics, or multiple params """
```

Comment:
Minor: I'm not sure I understand the docstring. I understand the method from reading it, set to index on 1-member long `topic` fields only. But I think the docstring could be better.

Reply:
How's "Do not index queries unless they consist of a single topic field" ?